Bottom: 1c452145c58e5771645421b6d9e5aaa3f39ac7b7
Top:    081d95eb5ac612b1f1136ba159a2ccb5716a75ee
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-09-29 23:12:00 +0100

Refresh of hid-gtk-add-missing-doxygen-co

---

diff --git a/src/hid/gtk/ghid-layer-selector.c b/src/hid/gtk/ghid-layer-selector.c
index 1afd211..432e57e 100644
--- a/src/hid/gtk/ghid-layer-selector.c
+++ b/src/hid/gtk/ghid-layer-selector.c
@@ -576,6 +576,7 @@ ghid_layer_selector_new (void)
  *  \param [in] color_string  The color of the layer on selector
  *  \param [in] visibile      Whether the layer is visible
  *  \param [in] selectable    Whether the layer appears in menus and can be selected
+ *  \param [in] renameable    Whether the layer is renameable
  */
 void
 ghid_layer_selector_add_layer (GHidLayerSelector *ls,
@@ -584,7 +585,7 @@ ghid_layer_selector_add_layer (GHidLayerSelector *ls,
                                const gchar *color_string,
                                gboolean visible,
                                gboolean selectable,
-                               gboolean builtin)
+                               gboolean renameable)
 {
   struct _layer *new_layer = NULL;
   gchar *pname, *vname;
@@ -652,7 +653,7 @@ ghid_layer_selector_add_layer (GHidLayerSelector *ls,
                       COLOR_COL,       color_string,
                       TEXT_COL,        name,
                       FONT_COL,        selectable ? NULL : "Italic",
-                      EDITABLE_COL,    !builtin,
+                      EDITABLE_COL,    renameable,
                       SELECTABLE_COL,  selectable,
                       SEPARATOR_COL,   FALSE,
                       -1);
diff --git a/src/hid/gtk/ghid-layer-selector.h b/src/hid/gtk/ghid-layer-selector.h
index 1ed5e01..c89112d 100644
--- a/src/hid/gtk/ghid-layer-selector.h
+++ b/src/hid/gtk/ghid-layer-selector.h
@@ -25,7 +25,7 @@ void ghid_layer_selector_add_layer (GHidLayerSelector *ls,
                                     const gchar *color_string,
                                     gboolean visible,
                                     gboolean selectable,
-                                    gboolean builtin);
+                                    gboolean renameable);
 
 gint ghid_layer_selector_install_pick_items (GHidLayerSelector *ls,
                                              GtkMenuShell *shell, gint pos);
diff --git a/src/hid/gtk/gui-top-window.c b/src/hid/gtk/gui-top-window.c
index 4f25e72..ee79f1a 100644
--- a/src/hid/gtk/gui-top-window.c
+++ b/src/hid/gtk/gui-top-window.c
@@ -799,22 +799,22 @@ make_virtual_layer_buttons (GtkWidget *layer_selector)
  
   layer_process (&color_string, &text, &active, LAYER_BUTTON_SILK);
   ghid_layer_selector_add_layer (layersel, LAYER_BUTTON_SILK,
-                                 text, color_string, active, TRUE, TRUE);
+                                 text, color_string, active, TRUE, FALSE);
   layer_process (&color_string, &text, &active, LAYER_BUTTON_RATS);
   ghid_layer_selector_add_layer (layersel, LAYER_BUTTON_RATS,
-                                 text, color_string, active, TRUE, TRUE);
+                                 text, color_string, active, TRUE, FALSE);
   layer_process (&color_string, &text, &active, LAYER_BUTTON_PINS);
   ghid_layer_selector_add_layer (layersel, LAYER_BUTTON_PINS,
-                                 text, color_string, active, FALSE, TRUE);
+                                 text, color_string, active, FALSE, FALSE);
   layer_process (&color_string, &text, &active, LAYER_BUTTON_VIAS);
   ghid_layer_selector_add_layer (layersel, LAYER_BUTTON_VIAS,
-                                 text, color_string, active, FALSE, TRUE);
+                                 text, color_string, active, FALSE, FALSE);
   layer_process (&color_string, &text, &active, LAYER_BUTTON_FARSIDE);
   ghid_layer_selector_add_layer (layersel, LAYER_BUTTON_FARSIDE,
-                                 text, color_string, active, FALSE, TRUE);
+                                 text, color_string, active, FALSE, FALSE);
   layer_process (&color_string, &text, &active, LAYER_BUTTON_MASK);
   ghid_layer_selector_add_layer (layersel, LAYER_BUTTON_MASK,
-                                 text, color_string, active, FALSE, TRUE);
+                                 text, color_string, active, FALSE, FALSE);
 }
 
 /*! \brief callback for ghid_layer_selector_update_colors */
@@ -848,7 +848,7 @@ make_layer_buttons (GtkWidget *layersel)
     {
       layer_process (&color_string, &text, &active, i);
       ghid_layer_selector_add_layer (GHID_LAYER_SELECTOR (layersel), i,
-                                     text, color_string, active, TRUE, FALSE);
+                                     text, color_string, active, TRUE, TRUE);
     }
 }
